data/campaigns/Legend_of_Wesmere/utils/low-macros.cfg

```diff
@@ -59,13 +59,16 @@
 #enddef
 
 #define ELVES_SCOUTS
-Elvish Scout, Elvish Rider, Elvish Outrider, Elvish Horse Archer#enddef
+Elvish Scout, Elvish Rider, Elvish Outrider
+#enddef
 
 #define ELVES_STEEL
-Elvish Fighter, Elvish Hero, Elvish Captain, Elvish Marshal, Elvish Champion, Elvish Archer, Elvish Marksman, Elvish Sharpshooter, Elvish Ranger, Elvish Avenger, {ELVES_SCOUTS}#enddef
+Elvish Fighter, Elvish Hero, Elvish Captain, Elvish Marshal, Elvish Champion, Elvish Archer, Elvish Marksman, Elvish Sharpshooter, Elvish Ranger, Elvish Avenger, {ELVES_SCOUTS}
+#enddef
 
 #define ELVES_NON_STEEL
-Elvish Shaman, Elvish Shyde, Elvish Enchantress, Elvish Sorceress, Elvish Sylph, Elvish Druid, Wose, Elder Wose, Ancient Wose, {ELVES_SCOUTS}#enddef
+Elvish Shaman, Elvish Shyde, Elvish Enchantress, Elvish Sorceress, Elvish Sylph, Elvish Druid, Wose, Elder Wose, Ancient Wose, {ELVES_SCOUTS}
+#enddef
```

Comment:
I'm not sure. The scenario 5's define already had an #endef on a new line, so it might not be expected to be used the same way? I don't know, I'll do what I should have done in the first place and change as little as possible.